dbt/project.py

```diff
@@ -40,6 +40,9 @@ def __repr__(self):
     def __getitem__(self, key):
         return self.cfg.__getitem__(key)
 
+    def __contains__(self, key):
+        return self.cfg.__contains__(key)
```

Comment:
@drewbanin semantically, is the config "the project" where as profiles are ancillary information? I don't have a good enough grasp on the code to know yet. If so, this seems reasonable, if not, explicitly accessing the `cfg` property of the project might be clearer below.